src/sync/pull.rs

```diff
@@ -798,15 +886,24 @@ mod tests {
                     last_mutation_id: 0,
                     ..good_pull_resp.clone()
                 }),
-                exp_err: Some("TimeTravel"),
                 exp_new_sync_head: None,
+                exp_begin_try_pull_result: Err(BeginTryPullError::TimeTravelProhibited(str!(
+                    "base lastMutationID 1 is > than client view lastMutationID 0; ignoring client view"
+                ))),
             },
             Case {
                 name: "pull 500s -> beginpull errors",
                 num_pending_mutations: 0,
                 pull_result: Err(str!("FetchNotOk(500)")),
```

Comment:
I guess this can be anything we want because we no longer have this specific error. "BOOM" would work just as well.

Reply:
Let me check...